Postgres/AppDelegate.m

```diff
@@ -89,16 +98,29 @@ - (void)applicationDidFinishLaunching:(NSNotification *)notification {
     [self.checkForUpdatesMenuItem setEnabled:YES];
     [self.checkForUpdatesMenuItem setHidden:NO];
 #endif
-        
+    
+    __weak AppDelegate *weakSelf = self;
+    [[NSDistributedNotificationCenter defaultCenter] addObserverForName:kAppleInterfaceThemeChangedNotification
+                                                                 object:nil
+                                                                  queue:[NSOperationQueue mainQueue]
+                                                             usingBlock:^(NSNotification *notification) {
+                                                                 BOOL darkMode = weakSelf.isDarkMode;
+                                                                 _templateOffImage.template = darkMode;
+                                                                 _templateOnImage.template = darkMode;
```

Comment:
Using an instance variable (`_templateOffImage`) causes a strong reference to self. To avoid the strong reference, you'd have to do something like `AppDelegate *strongSelf = weakSelf; if (strongSelf) {strongSelf->_templateOffImage.template=darkMode;}`